source/sass/blocks/page-footer.scss

```
@@ -1,12 +1,54 @@
 .page-footer {
+  padding-bottom: 23px;
   background-color: #ffffff;
 }
 
 .page-footer__title {
+  margin: 0;
+  margin-bottom: 37px;
+  padding: 0;
   font-size: 21px;
   line-height: 26px;
   font-weight: 700;
   color: #000000;
 
   text-transform: uppercase;
 }
+
+.page-footer__wrapper {
+  display: flex;
+  padding-top: 36px;
+  flex-direction: column;
+  align-items: center;
+}
+
+@media (min-width: $tablet-width) {
+  .page-footer {
+    padding-bottom: 29px;
+  }
+
+  .page-footer__wrapper {
+    flex-direction: row;
+    padding: 0;
+    padding-top: 25px;
+    padding-right: 60px;
+    padding-left: 60px;
```

Лучше так
```suggestion
    padding-top: 25px;
    padding-right: 60px;
    padding-bottom: 0;
    padding-left: 60px;
```
или так
```suggestion
    padding: 25px 60px 0 60px;
```